src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -660,6 +662,45 @@ executeCond opts sc cc cs ss = do
             (ss ^. MS.csFreshPointers)
   OM (setupValueSub %= Map.union ptrs)
 
+  liftIO . mapM_ print $ cs ^. MS.csPreState . MS.csAllocs
+
```

Comment:
Everything below here could be factored out into another function (along the lines of `execute*`)